src/components/ui/ColorPickerInput.js

```diff
@@ -0,0 +1,111 @@
+import { Component } from 'react';
```

Comment:
Can you please convert this file to typescript?

Reply:
Yes, thought about it. I'm still learning proper typescript so hopefully I can convert this file successfully. Will try yo do it as soon as I can. 😊

Reply:
Yes, thought about it. I'm still learning proper typescript so hopefully I can convert this file successfully. Will try to do it as soon as I can. 😊

Reply:
I have converted that file into typescript. Please review and run locally. If it's good, then please merge. Thanks!